internal/k8s/client.go

```diff
@@ -278,6 +285,15 @@ func (k K8sClient) actOnEntities(ctx context.Context, cmdArgs []string, entities
 	return k.kubectlRunner.execWithStdin(ctx, args, stdin)
 }
 
+func ProvideClientSet(cfg *rest.Config) (*kubernetes.Clientset, error) {
+	clientSet, err := kubernetes.NewForConfig(cfg)
+	if err != nil {
+		return nil, err
+	}
+
+	return clientSet, nil
+}
+
 func ProvideCoreInterface(cfg *rest.Config) (apiv1.CoreV1Interface, error) {
```

Comment:
let's update this to take a ClientSet, so that we don't have two independent ClientSets running around